src/test/java/BringItOn/test/AbstractTest.java

```diff
@@ -0,0 +1,50 @@
+package BringItOn.test;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
+import BringItOn.page.PastebinMainPage;
+import BringItOn.page.PastebinSubmittedPastePage;
+
+public abstract class AbstractTest {
+    protected WebDriver driver;
+
+    protected static final String NEW_PASTE_TEXT = "git config --global user.name  \"New Sheriff in Town\"\n" +
+            "git reset $(git commit-tree HEAD^{tree} -m \"Legacy code\")\n" +
+            "git push origin master --force";
+    protected static final String NEW_PASTE_TITLE = "how to gain dominance among developers";
+    protected static final String SUBMITTED_PASTE_TITLE_LOCATOR = "//div[@class='info-top']/h1";
+    protected static final String EXPECTED_PASTE_SYNTAX = "Bash";
+    PastebinMainPage pastebinMainPage;
+    PastebinSubmittedPastePage pastebinSubmittedPastePage;
```

Comment:
It used to be protected initially - here and in many other parts of the code. But since I don't know much about it yet, I decided to follow IDE suggestions on code changes and edited it everywhere it was suggested to avoid a single warning from IDE. I definitely need to go deeper into it to understand purposes and approaches of using different modifiers in Java.